packages/fxa-auth-server/lib/pushbox/index.ts

```diff
@@ -190,19 +242,20 @@ module.exports = function (log, config, statsd, DB = PushboxDB) {
      * @param {Object} data - data object to serialize into storage
      * @returns {Promise} object with `index` to the inserted record
      */
-    async store(uid, deviceId, data, ttl) {
+    async store(uid: string, deviceId: string, data: any, ttl: number) {
       if (typeof ttl === 'undefined' || ttl > maxTTL) {
         ttl = maxTTL;
       }
 
       if (shouldUseDb()) {
         const startTime = performance.now();
         try {
-          const result = await pushboxDb.store({
+          const result = await pushboxDb!.store({
             uid,
             deviceId,
             data: encodeForStorage(data),
-            ttl,
+            // ttl is in seconds
+            ttl: Math.ceil(Date.now() / 1000) + ttl,
```

Comment:
😅  Noticed this while reading through some code.  Would've caught it anyway during manual Push Tab test on local env.